newthread.php

```diff
@@ -1090,6 +1090,8 @@
 		}
 	}
 
+	$pollbox='';
```

Comment:
This should also follow MyBB Development Standards with additional spaces:
`$pollbox = '';`